test/PowerShellEditorServices.Test.E2E/LanguageServerProtocolMessageTests.cs

```diff
@@ -816,6 +816,32 @@ public async Task CanSendCompletionAndCompletionResolveRequest()
             Assert.Contains("Writes customized output to a host", updatedCompletionItem.Documentation.String);
         }
 
+        [Fact]
+        public async Task CanSGetHelpCompletionResolveWithPrefixRequest()
+        {
+            EvaluateResponseBody evaluateResponseBody =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'evaluateResponseBody'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5252933)